lib/everypolitician/popolo/event.rb

```diff
@@ -3,6 +3,20 @@ module Popolo
     class Events < Collection; end
     class Event < Entity
       attr_accessor :start_date, :end_date
+
+      def people
+        raise NotImplementedError unless classification == 'legislative period'
+        ids = popolo.memberships.select { |m| m.legislative_period_id == id }
```

Comment:
This should probably go through the public search interface, rather than rolling its own `select` [not least because then it'll gain any benefit we get from making searches not use a linear scan]